package.json

```diff
@@ -140,6 +140,7 @@
   },
   "packageManager": "yarn@4.3.1",
   "resolutions": {
+    "bootstrap-select": "1.13.6",
```

Comment:
Should this have `~`?

Reply:
Yeah we can add that, I repushed with the ~ added